### PR TITLE
fix: DV-2815 prepare for update for smaller docker images

### PR DESCRIPTION
**Pull request by joshughes**
Depends on: https://github.com/6RiverSystems/ci_scripts/pull/30 

In the end, helps reduce images size by up to 1 GB for every service built this way. 

We will no longer ship Dev dependencies so as we move over to this we need to test to ensure our package.json is in good shape.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,9 +1,12 @@
+orbs:
+  gcp-cli: circleci/gcp-cli@1.0.0
+
 version: 2.1
 
 jobs:
   install:
     docker:
-      - image: circleci/node:12.16.3
+      - image: cimg/node:12.16.3
```
Comment (mgabeler-lee-6rs) [inline .circleci/config.yml:9]: What's the difference between `circleci/node:...` vs `cimg/node:...`?

**Review by mgabeler-lee-6rs — COMMENTED** [review]
Just one question, otherwise LGTM

File: .circleci/config.yml (hunk 1 above)
Comment (joshughes) [reply .circleci/config.yml:9]: https://circleci.com/docs/2.0/circleci-images/#next-generation-convenience-images

Comment (mgabeler-lee-6rs) [reply .circleci/config.yml:9]: OK that doesn't really explain what's different other than it's public and they should be smaller/faster, but OK ... this is basically just "newer verison of circleci images" :)

**Review by mgabeler-lee-6rs — APPROVED** [review]

**Comment by mgabeler-lee-6rs** [comment]
I'd be curious to see the docker image before/after for this minimal repo, since it should give a baseline for the minimum possible benefit.

**Comment by joshughes** [comment]
```
REPOSITORY                                     TAG                 IMAGE ID            CREATED             SIZE
gcr.io/plasma-column-128721/template-service   2.1.1               c0bfbe0bc034        4 days ago          1.26GB
```

**Pull request merged**